.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+env:
+  GH_TOKEN: ${{ github.token }}      
+
+jobs:
+    release:
+        name: Create new release
+        runs-on: ${{ matrix.os }}
+        strategy:
+          matrix:
+            include:
+              - os: ubuntu-latest
+                filename_suffix: "${{ github.ref_name }}-linux-amd64"
+        steps:
+          - uses: actions/checkout@v4
+          - uses: dtolnay/rust-toolchain@stable
+          - run: cargo build --release --locked ${{ matrix.flags }}
```

Comment:
`${{ matrix.flags }}` is empty, right? Better remove it then.

Reply:
Yeah, seems like I was indeed *too tired*, this should've been removed.